notes/2024/2024-04-04.md

```diff
@@ -0,0 +1,83 @@
+# 2024-04-04ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2024-03-21-transcript.md`](2024-03-21-transcript.md)
+
+## Attending
+
+- Nicholas C. Zakas (@nzakas) - TSC
+- Milos Djermanovic (@mdjermanovic) - TSC
+- Francesco Trotta (@fasttime) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+### Status of previous actionable items
+
+- [x] [feat: support the new flat config](https://github.com/eslint/create-config/issues/51) (Completed)
+- [x] [Switch Playground to flat config](https://github.com/eslint/eslint.org/issues/507) (Completed)
+
+### [Change Request: `--inspect-config` CLI option](https://github.com/eslint/eslint/issues/18255)
+
+**TSC Summary:** This issue seeks to add a --inspect-config CLI option that will launch the config inspector (similar to --init does with @eslint/create-config).
+
+**TSC Question:** Shall we accept this proposal?
+
+**Resolution:** We've agreed to accept this proposal
+
+**Action Items:** @nzakas will work on the PR
+
+### [feat: Support different names for code block languages](https://github.com/eslint/eslint-plugin-markdown/issues/245)
```

Comment:
I am not able to remove the `tsc-agenda` label. Looks like I don't have the right permissions:

<img width="385" alt="Screenshot 2024-04-08 at 5 26 17 AM" src="https://github.com/eslint/tsc-meetings/assets/180152/7777b423-c40e-41c4-8875-b5d505904c92">

Reply:
@sam3k can you try again? You should have access now.

Reply:
@fasttime it works now! Thank you!